android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -137,6 +141,10 @@ private static void throwEncodeException(JSONException e) {
     throw new RuntimeException("Could not encode JSON", e);
   }
 
+  private static void deleteFile(Context context, String fileName) {
+    context.deleteFile(fileName);
```

Comment:
I was under the impression that didn't delete the file but only cleared the contents. Does Android automatically delete empty preference files?